torch/csrc/distributed/rpc/process_group_agent.cpp

```diff
@@ -50,10 +50,10 @@ double ProcessGroupAgent::AverageMetricsTracker::computeAverage() {
 const ProcessGroupAgent::steady_clock_time_point
     ProcessGroupAgent::kInfiniteTimeoutTimePoint =
         std::chrono::time_point<std::chrono::steady_clock>::max();
-const std::string kNumPendingRequests = "num_pending_requests";
-const std::string kThreadPoolSize = "thread_pool_size";
-const std::string kNumIdleThreads = "num_idle_threads";
-const std::string kGilAverageWaitTime = "gil_average_wait_time_us";
+const std::string kNumPendingRequests = "agent.num_pending_requests";
```

Comment:
@pritamdamania87 I'm trying to keep it the same as the metrics for the internal RPC Agent which just uses "agent" as well. Would it be better to rename both of them?

Reply:
I see, I guess "agent" makes sense since we would actually only use 1 agent at a time. Although, maybe we can rename "agent" to "rpc_agent" in a future diff for more clarity.